Panic in VPN Server Route deletion

<!--- Please keep this note for the community --->

### Community Note

* Please vote on this issue by adding a 👍 [[reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/)](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

<!--- Thank you for keeping this note for the community --->

### Terraform CLI and Terraform IBM Provider Version

Terraform v1.x.x
Provider version v1.80.4

### Affected Resource(s)

* ibm_is_vpn_server_route

### Terraform Configuration Files

```hcl
resource "ibm_is_vpn_server_route" "example" {
  vpn_server = ibm_is_vpn_server.example.id
  destination = "192.168.1.0/24"
  action = "translate"
  name = "example-route"
}
```

### Debug Output

N/A

### Panic Output

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x60 pc=0x4c7d9d9]
goroutine 263 [running]:
github.com/IBM-Cloud/terraform-provider-ibm/ibm/service/vpc.isWaitForVPNServerRouteDeleted.func1()
	github.com/IBM-Cloud/terraform-provider-ibm/ibm/service/vpc/resource_ibm_is_vpn_server_route.go:439 +0x319
github.com/hashicorp/terraform-plugin-sdk/v2/helper/retry.(*StateChangeConf).WaitForStateContext.func1()
	github.com/hashicorp/terraform-plugin-sdk/v2@v2.35.0/helper/retry/state.go:113 +0x1a3
```

### Expected Behavior

VPN server route should be deleted without provider crash.

### Actual Behavior

Provider crashes with nil pointer dereference during deletion.

### Steps to Reproduce

1. `terraform apply` to create VPN server route
2. `terraform destroy` to delete the resource
3. Provider crashes during deletion wait logic

### Important Factoids

NA
### References
NA

--- a/ibm/service/vpc/resource_ibm_is_vpn_server_route.go
+++ b/ibm/service/vpc/resource_ibm_is_vpn_server_route.go
@@ -415,8 +415,8 @@ func resourceIBMIsVPNServerRouteDelete(context context.Context, d *schema.Resour
 }
 
 func isWaitForVPNServerRouteDeleted(context context.Context, sess *vpcv1.VpcV1, d *schema.ResourceData) (interface{}, error) {
+	log.Printf("Waiting for VPN Server Route(%s) to be deleted.", d.Id())
 
-	log.Printf("Waiting for VPN Server  Route(%s) to be deleted.", d.Id())
 	stateConf := &resource.StateChangeConf{
 		Pending: []string{"retry", isVPNServerRouteStatusDeleting},
 		Target:  []string{isVPNServerStatusDeleted, isVPNServerRouteStatusFailed},
@@ -436,7 +436,7 @@ func isWaitForVPNServerRouteDeleted(context context.Context, sess *vpcv1.VpcV1, 
 				if response != nil && response.StatusCode == 404 {
 					return vpnServerRoute, isVPNServerRouteStatusDeleted, nil
 				}
-				return vpnServerRoute, *vpnServerRoute.LifecycleState, flex.TerraformErrorf(err, fmt.Sprintf("DeleteVPNServerRouteWithContext failed: %s", err.Error()), "ibm_is_vpn_server_route", "delete-wait")
+				return vpnServerRoute, isVPNServerRouteStatusFailed, flex.TerraformErrorf(err, fmt.Sprintf("GetVPNServerRouteWithContext failed: %s", err.Error()), "ibm_is_vpn_server_route", "delete-wait")
 			}
 			return vpnServerRoute, *vpnServerRoute.LifecycleState, nil
 

